src/server.tsx

```diff
@@ -58,25 +57,28 @@ app.get('/healthcheck', (req: express.Request, res: express.Response) => {
 interface Epic {
     html: string;
     css: string;
+    testName: string;
+    variantName: string;
 }
 
-const fiveMinutes = 60 * 5;
-const [, fetchDefaultEpicContentCached] = cacheAsync(fetchDefaultEpicContent, fiveMinutes);
+const [, fetchConfiguredEpicTestsCached] = cacheAsync(fetchConfiguredEpicTests, 60);
```

Comment:
Possibly worth optimising this caching time? I know the strategy depends on the lifespan of the instance itself but only 1 minute seems low 🤔

Reply:
That's definitely a valid point, but I think better to change in a separate PR if that sounds okay?

Reply:
Sure